packages/functional-tests/src/tests/physics-bounce-test.ts

```diff
@@ -7,21 +7,21 @@ import * as MRE from '@microsoft/mixed-reality-extension-sdk';
 
 import { Test } from '../test';
 
-const defaultBallColor = MRE.Color3.FromInts(220, 150, 150);
-
 export default class PhysicsBounceTest extends Test {
 	public expectedResultDescription = "Balls and boxes hit the ground and bounce.";
 	private assets: MRE.AssetContainer;
 	private interval: NodeJS.Timeout;
-	private ballboxMat: MRE.Material;
+	private materials: MRE.Material[] = [];
 	private bouncePlane: MRE.Actor;
 
 	public async run(root: MRE.Actor): Promise<boolean> {
 		this.assets = new MRE.AssetContainer(this.app.context);
 
-		this.ballboxMat = this.assets.createMaterial('ball', {
-			color: defaultBallColor
-		});
+		this.materials.push(this.assets.createMaterial('mat1', { color: MRE.Color3.FromHexString('#ff0000').toColor4() }));
```

Comment:
I am getting here the error " This line has a length of 123. Maximum allowed is 120  max-len". Can this length be adjusted in VS?